### ISSUE TITLE
WCAG test

### ISSUE THREAD

**Issue by MathijsN** [MathijsN/look-and-feel-corporate-identity#17]
Lighthouse test:

<img width="819" height="708" alt="Image" src="https://github.com/user-attachments/assets/9fa6c50b-11fd-41de-877c-10161e86b116" />

Everything is reachable using tab.

For color contrast, these images should be darker so the text will be easier to read.

<img width="472" height="373" alt="Image" src="https://github.com/user-attachments/assets/d41c0472-f8d5-4ff8-840e-427f16c433b9" />
